sensor_control/navigator_lidar_oa/src/VolumeClassifier.h

```diff
@@ -0,0 +1,21 @@
+#include "ConnectedComponents.h"
+#include "navigator_msgs/PerceptionObject.h"
+
+void VolumeClassifier(objectMessage &object)
+{
+	//If we classify something as shooter/scanthecode, leave it alone
+	if (object.name == navigator_msgs::PerceptionObject::START_GATE_BUOY || object.name == navigator_msgs::PerceptionObject::SCAN_THE_CODE) {
+		return;
+	}
+	//if (object.name == navigator_msgs::PerceptionObject::BUOY || object.name == navigator_msgs::PerceptionObject::UNKNOWN) {
+		if (object.scale.x > 4 && object.scale.y > 4 && object.scale.z > 2 && object.strikesPersist.size() > 600 ) {
+			object.name = navigator_msgs::PerceptionObject::DETECT_DELIVER_PLATFORM;
+		} else if (object.scale.x > 1.5 && object.scale.y > 1.5 && object.scale.z > 1.5 && object.strikesPersist.size() > 300) {
+			object.name = navigator_msgs::PerceptionObject::TOTEM;
+		} else if (false) {
+			object.name = navigator_msgs::PerceptionObject::SCAN_THE_CODE;
```

Comment:
Andy discussed this morning about verifying the height of the competition scan the code structure and making ours match. We'll be able to pick bounds then! Until then, we can set the name manually.